src/main/java/com/thangle/api/user/UserDTO.java

```diff
@@ -0,0 +1,22 @@
+package com.thangle.api.user;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class UserDTO {
+    private UUID id;
+    private String username;
+    private String password;
```

Comment:
We won't response user's password to client. Even return null. We should remove the field to the model